src/EnergyPlus/PlantUtilities.cc

```diff
@@ -573,21 +574,29 @@ namespace PlantUtilities {
 
 			Real64 const a_node_MasFlowRate( a_node.MassFlowRate );
 			Real64 const a_node_MasFlowRateRequest( a_node.MassFlowRateRequest );
+			EMSLoadOverride = false;
+			// check to see if any component on branch uses EMS On/Off Supervisory control to shut down flow
+			for ( int CompNum = 1, CompNum_end = branch.TotalComponents; CompNum <= CompNum_end; ++CompNum ) {
+				auto const & comp( branch.Comp( CompNum ) );
+				if ( comp.EMSLoadOverrideOn ) EMSLoadOverride = true;
```

Comment:
Doesn't this need to check the value as well?   ( comp.EMSLoadOverrideValue == 0.0 )